metadata.rb

```diff
@@ -16,7 +16,7 @@
 supports 'oracle'
 
 depends 'build-essential'
-depends 'mysql', '>= 6.0'
+depends 'mysql',               '~> 8.0.1'
```

Comment:
This is a very tight pin; it’d be better to set this to `~> 8.0` to allow all non-breaking changes (i.e., what happens when mysql v8.1.0 ships?).